### PR TITLE
docs: how to check resource status

### PR DESCRIPTION
**Pull request by gxthrj**
ref to : https://github.com/apache/apisix-ingress-controller/issues/537

### CODE REVIEW

**Comment by tokers** [comment]
@gxthrj Please fill some issue description.

File: docs/en/latest/practices/check-crd-status.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+When using the APISIX ingress declarative configuration, often use the `kubectl apply` command. If the configuration is verified by the schema and validation webhook, the configuration will be accepted by Kubernetes.
```
Comment (tokers) [inline docs/en/latest/practices/check-crd-status.md:24]:
```suggestion
When using the APISIX ingress declarative configuration, often use the `kubectl apply` command. Only if the configuration was verified by its [Open API V3 Schema definition](https://swagger.io/specification/) and its validation webhooks (if any), can the configuration be accepted by Kubernetes.
```

File: docs/en/latest/practices/check-crd-status.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+When the Ingress Controller watches the resource change, the logic unit of the Ingress Controller has just started to work.
```
Comment (tokers) [inline docs/en/latest/practices/check-crd-status.md:26]: Use term Apache APISIX Ingress Controller.

File: docs/en/latest/practices/check-crd-status.md (hunk 1 above)
Comment (gxthrj) [reply docs/en/latest/practices/check-crd-status.md:24]: OK

File: docs/en/latest/practices/check-crd-status.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+When using the Apache APISIX Ingress Controller declarative configuration, often use the `kubectl apply` command. Only if the configuration was verified by its [Open API V3 Schema definition](https://swagger.io/specification/) and its validation webhooks (if any), can the configuration be accepted by Kubernetes.
+
+When the Apache APISIX Ingress Controller watches the resource change, the logic unit of the Apache APISIX Ingress Controller has just started to work. 
+In various operations of the Apache APISIX Ingress Controller, object conversion and more verification will be performed. 
+When the verification fails, the Apache APISIX Ingress Controller will throw an error message and will continue to retry
```
Comment (tao12345666333) [inline docs/en/latest/practices/check-crd-status.md:28]:
```suggestion
When the verification fails, the Apache APISIX Ingress Controller will log an error message and will continue to retry 
```

File: docs/en/latest/practices/check-crd-status.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+e.g.
+   
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: apisix.apache.org/v2alpha1
+kind: ApisixRoute
+metadata:
+  name: httpbin-route
+  namespace: test
```
Comment (tao12345666333) [inline docs/en/latest/practices/check-crd-status.md:54]: If namespace is specified here, you need to create it first.

File: docs/en/latest/practices/check-crd-status.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+When using the Apache APISIX Ingress Controller declarative configuration, often use the `kubectl apply` command. Only if the configuration was verified by its [Open API V3 Schema definition](https://swagger.io/specification/) and its validation webhooks (if any), can the configuration be accepted by Kubernetes.
+
+When the Apache APISIX Ingress Controller watches the resource change, the logic unit of the Apache APISIX Ingress Controller has just started to work. 
+In various operations of the Apache APISIX Ingress Controller, object conversion and more verification will be performed. 
+When the verification fails, the Apache APISIX Ingress Controller will throw an error message and will continue to retry 
+until the declared state is successfully synchronized to APISIX.
+
+Therefore, after the declarative configuration is accepted by Kubernetes, it does not mean that the configuration is synchronized to APISIX.
+
+In this practice, we will show how to  check the status of CRD.
```
Comment (tokers) [inline docs/en/latest/practices/check-crd-status.md:33]:
```suggestion
In this practice, we will show how to check the status of CRD.
```

File: docs/en/latest/practices/check-crd-status.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+## Prerequisites
+
+- an available Kubernetes cluster
```
Comment (tokers) [inline docs/en/latest/practices/check-crd-status.md:37]: We may add some version hints.

**Review by tokers — APPROVED** [review]

File: docs/en/latest/practices/check-crd-status.md (hunk 4 above)
Comment (gxthrj) [reply docs/en/latest/practices/check-crd-status.md:54]: Namespapce removed.

**Review by tokers — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]
LGTM 
should fix markdown lint CI .

**Review by tao12345666333 — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**